ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,200 @@
+                "nterdum. Nunc eu vestibulum nisi, at luctus est. Ut eget pretium sapien. Nullam egestas, ipsum at placerat euismod," +
+                " turpis quam convallis mauris, et condimentum libero mi sit amet nulla. Pellentesque in purus sit amet neque ultric" +
+                "ies mattis eget ut nisl. Phasellus eget ultrices risus. Suspendisse dapibus neque vitae scelerisque efficitur. Susp" +
+                "endisse rhoncus, ligula ac ullamcorper sodales, diam arcu varius ipsum, vel congue nisi urna quis nibh. Aliquam eleif" +
+                "end dui urna, ut euismod ligula aliquam eget. Vestibulum tincidunt quam dolor, eu convallis justo finibus eget. Curabi" +
+                "tur eget tristique nisi, at pellentesque nunc. In bibendum consectetur condimentum. Ut tincidunt, lectus sed rut" +
+                "rum lacinia, sapien diam accumsan sapien, sed mollis mauris sapien ut lectus. Pellentesque fringilla sem eget neque fe" +
+                "rmentum efficitur. Fusce dignissim metus eu purus vulputate vestibulum. Cras at gravida sapien. Morbi non tincidun" +
+                "t leo. Sed justo odio, bibendum ut elit.";
+            // oh god
+            LoremIpsumGenerator(LoremTest);
+        }
+
+        public static bool Compare(string str1, string str2)
+        {
+            if (str1.Length != str2.Length)
+                return false;
+            for (int i = 0; i < str1.Length; i++)
+            {
+                if (str1[i] != str2[i])
+                    return false;
+            }
+            return true;
+
+        }
+        public static void Analyze(string str, out int alpha, out int digits, out int special)
+        {
+            alpha = 0;
+            digits = 0;
+            special = 0;
+
+            for (int i = 0; i < str.Length; i++)
+            {
+                if (char.IsLetter(str[i]))
+                    alpha++;
+                else if (char.IsDigit(str[i]))
+                    digits++;
+                else
+                    special++;
+            }
+        }
+
+        public static string Sort(string str)
+        {
+            str.ToLower();
+            char[] sortedC = new char[str.Length];
+            for (int i = 0; i < str.Length; i++) 
+            {
+                sortedC[i] = str[i];
+            }
+            for (int i = 0; i < sortedC.Length; i++)
+            {
+                for (int j = 0; j < sortedC.Length - 1; j++)
+                {
+                    if (sortedC[j] > sortedC[j + 1])
+                    {
+                        char t = sortedC[j];
+                        sortedC[j] = sortedC[j + 1];
+                        sortedC[j + 1] = t;
+                    }
+                }
+            }
+            string sorted = new string(sortedC);
+            return sorted;
+        }
+        public static char[] Duplicate(string str)
```

Comment:
your solution not working for next test case
Input -> Hello and hi 
Outpur -> ['h', 'l']